.github/workflows/frontend-qa.yml

```
@@ -33,6 +33,30 @@ jobs:
       - name: Run lint
         run: pnpm lint
 
+      - name: Run component tests
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)